content/docs/command-reference/cache/dir.md

```diff
@@ -18,7 +18,7 @@ positional arguments:
 ## Description
 
 Helper to set the `cache.dir` configuration option. (See
-[cache directory](/doc/user-guide/dvc-files-and-directories#structure-of-the-cache-directory).)
+[cache directory](/doc/user-guide/glossary/dvc-cache#structure-of-the-cache-directory).)
```

Comment:
Can anchored links have their own search history with Google @jeremydesroches ?

I'm thinking whether and how to setup a redirect for this change... Seems tricky since the old content will be split into a new page.

Reply:
> Can anchored links have their own search history with Google @jeremydesroches ?

No, they can't. Depending on content and query an #id anchor and subtext might be used for a snippet, but that's true for all structured content.

So we don't need a redirect on the #id anchors. Only for any base URLs that we remove or change. (Plus it doesn't really work like that... would redirect all `.../files-and-directories` links).

Reply:
> So we don't need a redirect

Reply:
> it doesn't really work like that

True...

> So we don't need a redirect

OK! But how do we best dodge losing traffic for removing half of the content from there? I guess adding a prominent link from the guide to the new concepts page? Could be removed later, when someone remembers.

Reply:
> how do we best dodge losing ranking/traffic for moving half of the content out of the guide?

If the content were totally removed there could be an issue. In this case, content updates are encouraged and moving it to another location on the site is not likely to be penalized. The search engine will notice and index the new content. It may even notice the similarity (a black box to us, though). 

When it is reindexed at the new location it should retain its original "value" as a starting point (same content). Assuming our reorganization and addition of concepts is an improvement (i.e., better content structure and engagement) then it will be able to grow on that basis.

>  I guess adding a prominent link from there to the new concepts page?

Yes, exactly, and any link will do in terms of making sure the search engine finds it and reindexes. There will be multiple links, it's in the nav, and it will show up in the sitemap, which is indexed often for DVC.org.

Reply:
Sounds good. In any case , per my recent https://github.com/iterative/dvc.org/pull/1944#discussion_r536392345, let's not move the details about the file formats to the concepts section (if it hasn't been done).

Reply:
> let's not move the details about the file formats to the concepts section

I see we haven't yet. OK. Resolving this.